<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleWidget</class>
 <widget class="QWidget" name="TitleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>41</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="Icon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Title"/>
   </item>
   <item>
    <spacer name="leftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="leftLayout"/>
   </item>
   <item>
    <spacer name="centralSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rightLayout"/>
   </item>
   <item>
    <spacer name="rightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="viewButton">
     <property name="text">
      <string>Dock</string>
     </property>
     <property name="icon">
      <iconset theme="go-up-symbolic"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="window-close-symbolic">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>TitleWidget</receiver>
   <slot>CloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>TitleWidget</receiver>
   <slot>ViewButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CloseButtonClicked()</slot>
  <slot>ViewButtonClicked()</slot>
 </slots>
</ui>
